cli/up/up.go

```diff
@@ -161,9 +151,9 @@ func (cmd *up) run(services []string) error {
 	stClient := cmd.makeSyncthingClient(parsedCompose)
 	idPathMap := stClient.GetIDPathMap()
 
-	regCreds, err := docker.GetLocalRegistryCredentials(cmd.dockerConfig)
+	regCreds, err := auth.GetLocalRegistryCredentials(cmd.dockerConfig)
 	if err != nil {
-		log.WithError(err).Debug("Failed to get local registry credentials. Private images will fail to pull.")
```

Comment:
Ooh I think this was Debug because it was spewing scary messages. Could you check the git history and see if we should leave it at Debug?

Reply:
Yep, https://github.com/kelda-inc/blimp/commit/b012b43efbcd39af5ec005b8cd64afd973b43b00#diff-b8b4740c0566b86202dfe38e33f2fa34L179-R179.